modularhistory/config/_debug_toolbar.py

```diff
@@ -11,20 +11,15 @@
 from django.conf import settings
 from django.http import HttpRequest
 
-from modularhistory.constants.environments import Environments
-
 
 def show_toolbar(request: HttpRequest) -> bool:
     """Determine whether to display the debug toolbar."""
     conditions = (
-        settings.DEBUG
-        and request.META.get('REMOTE_ADDR', None) in settings.INTERNAL_IPS,
+        settings.DEBUG and request.META.get('REMOTE_ADDR', None) in settings.INTERNAL_IPS,
         request.user.is_superuser,
     )
     disqualifiers = (
-        settings.TESTING,
-        # disable toolbar in Dockerized dev environment to avoid errors; TODO: fix?
-        settings.ENVIRONMENT == Environments.DEV and settings.DOCKERIZED,
+        [settings.TESTING]
```

Comment:
I think this should be:
```
disqualifiers = (settings.TESTING,)
```
or
```
disqualifiers = (
    settings.TESTING,
)
```

Reply:
I was getting `TypeError: 'bool' object is not iterable` errors, didn't know adding a comma `,` would fix that.

Reply:
Ah, yes, the comma is how Python knows that it's a tuple even though the parentheses only contain one item.